opts/opts.go

```diff
@@ -308,6 +307,16 @@ func ValidateSysctl(val string) (string, error) {
 	return "", fmt.Errorf("sysctl '%s' is not whitelisted", val)
 }
 
+func ValidateProgressOutput(val string) (string, error) {
```

Comment:
Do you really need to return a string here?

Reply:
I think it's to respect the other validation `Validate*` signatures, that are able to mutate the value being validated. As we don't really use it into a type that require this signature (and as I'm not found of a validation function mutating the value it validates), I think we shouldn't return a `string` and just an error